prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -87,6 +95,40 @@ object FhirPathUtils : Logging {
         bundle: Bundle,
         expressionNode: ExpressionNode
     ): String {
-        return defaultPathEngine.evaluateToString(appContext, focusResource, bundle, bundle, expressionNode)
+        val evaluated = defaultPathEngine.evaluate(appContext, focusResource, bundle, bundle, expressionNode)
+        if (evaluated.size == 0) return ""
+        if (evaluated.size > 1 || !evaluated[0].isPrimitive) {
+            // Should only be evaluating primitives here
+            throw IllegalStateException()
```

Comment:
When throwing an exception always add a message.  On top of that, do we need to log anything or is it logged somewhere else?